### PR TITLE
Autocropping versus svg

### PR DESCRIPTION
**Pull request by boutell**
## Summary

The manual cropping interface does not appear for SVGs, whether the placement involves a fixed aspectRatio or not.

The autocrop feature should behave analogously by just not trying to autocrop SVGs.

(Not allowing them to be selected in the presence of aspectRatio is an interesting idea but not in scope and not required for A2 parity.)

## What are the specific steps to test this change?

* Fire up the `feature-image-cropping` branch of `testbed` with the `autocropping-versus-svg` branch of apostrophe
* Verify you can pick an svg for an image widget nested in a two-column widget (has 3x2 aspect ratio constraint). The svg will not be constrained, that's OK
* Verify that if you pick a jpg it does get constrained when you save the widget, even if you never manually crop it (this is autocrop)

## What kind of change does this PR introduce?
*(Check at least one)*

- [X] Bug fix
- [ ] New feature
- [ ] Refactor
- [ ] Documentation
- [ ] Build-related changes
- [ ] Other

## Make sure the PR fulfills these requirements:

- [X] It includes a) the existing issue ID being resolved, b) a convincing reason for adding this feature, or c) a clear description of the bug it resolves
- [ ] The changelog is updated
- [ ] Related documentation has been updated
- [ ] Related tests have been updated

If adding a new feature without an already open issue, it's best to open a **feature request issue** first and wait for approval before working on it.

**Other information:**

### CODE REVIEW

File: modules/@apostrophecms/attachment/index.js [hunk 1]
```diff
@@ -467,7 +467,9 @@ module.exports = {
         }
 
         if (!self.croppable[info.extension]) {
-          throw new Error(info.extension + ' files cannot be cropped, do not present cropping UI for this type');
+          throw self.apos.error('invalid', req.t('apostrophe:fileTypeCannotBeCropped', {
```
Comment (boutell) [inline modules/@apostrophecms/attachment/index.js:470]: You'll never see this (it's a sanity check for a situation the UI should avoid) but I wanted to format it properly anyway. I was seeing it until I fixed autocrop to avoid it.

File: modules/@apostrophecms/image/ui/apos/components/AposImageRelationshipEditor.vue [hunk 2]
```diff
@@ -269,7 +269,8 @@ export default {
           body: {
             _id: this.item.attachment._id,
             crop: this.docFields.data
-          }
+          },
+          busy: true
```
Comment (boutell) [inline modules/@apostrophecms/image/ui/apos/components/AposImageRelationshipEditor.vue:273]: Unrelated fix: block the UI when saving in the manual cropping interface (crop generation takes a moment)

**Review by myovchev — APPROVED** [review]
